portfolio/src/main/webapp/index.html

```diff
@@ -32,7 +32,14 @@
     <script src="request.js"></script>
     <script src="calendarAPI.js"></script>
     <script src="sheetsApi.js" onload="handleClientLoad()"></script>
-    <script src="https://apis.google.com/js/api.js"></script>
+    <script src="tasksApi.js"></script>
+    <script
+      async
+      defer
+      src="https://apis.google.com/js/api.js"
+      onload="this.onload=function(){};handleClientLoadTasks()"
```

Comment:
Consider adding some HTML comment on this onload and onreadystatechange logic. It wasn't immediately clear to me for example why you wrote this.onload=function(){};

Reply:
I removed this since my code worked without it. I had gotten it from a quick start example.